dc-cudami-editor/src/components/IdentifiableListItem.jsx

```diff
@@ -27,80 +29,82 @@ const IdentifiableListItem = ({
   const {t} = useTranslation()
   const viewUrl = `${apiContextPath}${type.toLowerCase()}s/${uuid}`
   return (
-    <ListGroupItem className="list-item pb-0 pt-0">
-      <Row>
-        <Col className="border-right pb-1 pt-1 text-right" md="1">
-          {index}
-        </Col>
-        <Col className="border-right pb-1 pt-1 text-center" md="1">
-          <figure className="mb-0">
-            <img
-              alt={altText?.[uiLocale] ?? ''}
-              src={
-                previewImage
-                  ? getImageUrl(previewImage, `${previewImageWidth},`)
-                  : `${apiContextPath}images/no-image.png`
-              }
-              style={{maxWidth: `${previewImageWidth}px`}}
-              title={title?.[uiLocale]}
-            />
-            {caption?.[uiLocale] && (
-              <figcaption>{caption[uiLocale]}</figcaption>
-            )}
-          </figure>
-        </Col>
-        <Col md="7" className="border-right pb-1 pt-1">
-          <a href={viewUrl}>{label}</a>
-        </Col>
-        <Col className="border-right pb-1 pt-1 text-center" md="2">
-          {new Date(lastModified).toLocaleString(uiLocale, {hour12: false})}
-        </Col>
-        <Col className="pb-1 pt-1 text-center" md="1">
-          <ButtonGroup>
+    <tr>
+      <td className="text-right">
+        {index}
+      </td>
+      <td className="text-center">
+        <figure className="mb-0">
+          <img
+            alt={altText?.[uiLocale] ?? ''}
+            src={
+              previewImage
+                ? getImageUrl(previewImage, `${previewImageWidth},`)
+                : `${apiContextPath}images/no-image.png`
+            }
+            style={{maxWidth: `${previewImageWidth}px`}}
+            title={title?.[uiLocale]}
+          />
+          {caption?.[uiLocale] && <figcaption>{caption[uiLocale]}</figcaption>}
+        </figure>
+      </td>
+      <td>
+        <a href={viewUrl}>{label}</a>
+      </td>
+      <td>
+        <ul className="list-inline mb-0">
+          {identifiers.map(({id, namespace}, index) => (
+            <li className="list-inline-item" key={index}>{`${
```

Comment:
Den Index im Array als `key` zu Verwenden ist ein Anti-Pattern vor dem React eigentlich im Dev-Modus auch warnt, besser etwas nehmen das das Item wirklcih eindeutig auszeichnet, wie in diesem Fall `id`.

Reply:
fehler meinerseits, hatte ich vorgeschlagen. war vorher besser mit namespace:id. sorry matthias

Reply:
Hab es wieder rückgängig gemacht und auch noch an ein paar anderen Stellen korrigiert.